Source/GTMSessionFetcherLogViewController.m

```diff
@@ -234,10 +254,10 @@ - (instancetype)initWithURL:(NSURL *)htmlURL
 }
 
 - (void)loadView {
-  UIWebView *webView = [[UIWebView alloc] init];
+  GTMWebView *webView = [[GTMWebView alloc] init];
   webView.autoresizingMask = (UIViewAutoresizingFlexibleWidth
                               | UIViewAutoresizingFlexibleHeight);
-  webView.delegate = self;
+  webView.navigationDelegate = self;
```

Comment:
Doesn't this need an `#if` to use `delegate` or `navigationDelegate`?

Reply:
Yup. Thanks for catching!